fe1-web/src/features/home/screens/Launch.tsx

```diff
@@ -106,10 +104,6 @@ const Launch = () => {
           <PoPTextButton onPress={connectToTestLao}>
             [TEST] Connect to LocalMockServer.ts (use &apos;npm run startServer&apos;)
           </PoPTextButton>
```

Comment:
Is it still useful? If it is the case, it might be nice to add it to `strings.ts`. I honestly never used this button, so I don't know if it's still relevant.

Reply:
Haha me neither! I guess we can remove it then:) (Together with LocalMockServer.ts) But I think I'll do that in a separate PR?